app/views/tags/index.html.erb

```diff
@@ -54,7 +54,7 @@
         <% end %>
 
         <% if tag.badge_id && tag.badge %>
-          <img src="<%= optimized_image_url(tag.badge.badge_image_url, width: 180) %>" style="width: 64px; height: 64px; transform: rotate(<%= -25 + (tag.id.digits.first * 5) %>deg);" class="right-1 bottom-1 absolute" />
+          <img src="<%= optimized_image_url(tag.badge.badge_image_url, width: 180) %>" style="width: 64px; height: 64px; transform: rotate(<%= -25 + (tag.id.digits.first * 5) %>deg);" class="right-1 bottom-1 absolute" alt="" />
```

Comment:
Wouldn't it make sense to make it like alt="<%= tag %>'s icon" or something?

Reply:
I concluded the image was decorative, since the tag is described in text previously in the DOM. Here's a screenshot where the JS logo follows the tag and follow button:

<img width="419" alt="JavaScript tag with heading, text, button, and logo" src="https://user-images.githubusercontent.com/1045233/101078358-e6c60d00-355a-11eb-8597-5f0a5cee5e1e.png">